### PR TITLE
Fix #3816 - add note about *-EventLog cmdlets

### LINKED ISSUE #3816: Add *-EventLog removal to PowerShell Core breaking changes
Issue Details
-----------------
The [Breaking Changes for PowerShell 6.0](https://docs.microsoft.com/en-us/powershell/scripting/whats-new/breaking-changes-ps6?view=powershell-6) article is missing that *-EventLog is removed with *-WinEvent as the replacement.

Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [x] Impacts 6 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not selecting all version of documents
----------------------------------------------------
- [x] The documented feature was introduced in selected version of PowerShell
- [ ] This issue only shows up in selected version of the document

### PR DESCRIPTION
**Pull request by sdwheeler**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [x] Impacts 6 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

File: reference/docs-conceptual/whats-new/breaking-changes-ps6.md [hunk 1]
```diff
@@ -66,6 +69,12 @@ PowerShell Core until a better solution is found.
 Due to the use of unsupported APIs, the `*-Counter` has been removed from PowerShell Core until a
 better solution is found.
 
+### `*-EventLog` cmdlets
+
+Due to the use of unsupported APIs, the `*-EventLog` has been removed from PowerShell Core. until a
+better solution is found. 'Get-WinEvent' and 'Create-WinEvent' are available to get and create
```
Comment (DCtheGeek) [inline reference/docs-conceptual/whats-new/breaking-changes-ps6.md]: Both cmdlets should be code fenced.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[0d53f12](https://github.com/sdwheeler/PowerShell-Docs/pull/3848/commits/0d53f12b8cd4500ed470861b663b7447e2546425)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/whats-new/breaking-changes-ps6.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3816/reference/docs-conceptual/whats-new/breaking-changes-ps6.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/breaking-changes-ps6?view=powershell-3.0&branch=pr-en-us-3848) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/breaking-changes-ps6?view=powershell-4.0&branch=pr-en-us-3848) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/breaking-changes-ps6?view=powershell-5.0&branch=pr-en-us-3848) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/breaking-changes-ps6?view=powershell-5.1&branch=pr-en-us-3848) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/breaking-changes-ps6?view=powershell-6&branch=pr-en-us-3848) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C11%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201903112136297293-3848%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ookra5v69jSdBE0nFuc32iROWQoNYtlrMxLi1zd1PcI%3D&st=2019-03-11T21%3A36%3A21Z&se=2019-04-11T21%3A41%3A21Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/docs-conceptual/whats-new/breaking-changes-ps6.md (hunk 1 above)
Comment (sdwheeler) [reply reference/docs-conceptual/whats-new/breaking-changes-ps6.md]: Doh, wrong character.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[5860174](https://github.com/sdwheeler/PowerShell-Docs/pull/3848/commits/58601740501e526802c7cd991fd339eda7c23d9f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/whats-new/breaking-changes-ps6.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3816/reference/docs-conceptual/whats-new/breaking-changes-ps6.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/breaking-changes-ps6?view=powershell-3.0&branch=pr-en-us-3848) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/breaking-changes-ps6?view=powershell-4.0&branch=pr-en-us-3848) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/breaking-changes-ps6?view=powershell-5.0&branch=pr-en-us-3848) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/breaking-changes-ps6?view=powershell-5.1&branch=pr-en-us-3848) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/breaking-changes-ps6?view=powershell-6&branch=pr-en-us-3848) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C11%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201903112327572857-3848%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ilhh560lOnQ6dNmLLwJ6h2oI%2B9GnY%2BG1UaWWQ6LFvoI%3D&st=2019-03-11T23%3A28%3A28Z&se=2019-04-11T23%3A33%3A28Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[d4111f9](https://github.com/sdwheeler/PowerShell-Docs/pull/3848/commits/d4111f9dfd9e04a64204dd4cc27fd516208379ba)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/whats-new/breaking-changes-ps6.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3816/reference/docs-conceptual/whats-new/breaking-changes-ps6.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/breaking-changes-ps6?view=powershell-3.0&branch=pr-en-us-3848) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/breaking-changes-ps6?view=powershell-4.0&branch=pr-en-us-3848) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/breaking-changes-ps6?view=powershell-5.0&branch=pr-en-us-3848) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/breaking-changes-ps6?view=powershell-5.1&branch=pr-en-us-3848) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/breaking-changes-ps6?view=powershell-6&branch=pr-en-us-3848) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C11%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201903112328225731-3848%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=E5yuawt9RnfXlW9IgCS9PYGFMhTAo3DA3eiyr9dZBnc%3D&st=2019-03-11T23%3A33%3A52Z&se=2019-04-11T23%3A38%3A52Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**